torch/testing/_internal/common_nn.py

```diff
@@ -5951,7 +5951,7 @@ def __call__(self, test_case):
             ref_input = deepcopy(input)
             ref_module = deepcopy(module)
             expected_out = self.reference_fn(ref_input, test_case._get_parameters(module)[0], ref_module)
-            test_case.assertEqual(out, expected_out)
+            test_case.assertEqual(out, expected_out.to(torch.double))
```

Comment:
Is this always double? This is a function that is used for different cases, probably better to use `exact_dtype=False` here instead of converting to double.